src/APL/vercel-apl.ts

```diff
@@ -55,27 +55,28 @@ export type VercelAPLConfig = {
  *   - SALEOR_DEPLOYMENT_TOKEN: token for your particular Vercel deployment
  */
 export class VercelAPL implements APL {
-  private registerAppURL: string;
+  private registerAppURL?: string;
 
-  private deploymentToken: string;
+  private deploymentToken?: string;
 
   constructor(config?: VercelAPLConfig) {
     const registerAppURL =
       config?.registerAppURL || process.env[VercelAPLVariables.SALEOR_REGISTER_APP_URL];
-    if (!registerAppURL) {
-      throw new Error("Misconfiguration: please provide registerAppUrl");
-    }
+
     const deploymentToken =
       config?.deploymentToken || process.env[VercelAPLVariables.SALEOR_DEPLOYMENT_TOKEN];
-    if (!deploymentToken) {
-      throw new Error("Misconfiguration: please provide deploymentToken");
-    }
```

Comment:
@krzysztofwolski I have no better idea at the moment. Later the error will be thrown. Maybe we can do it like this, later we can think of something better. I don't want to overcomplicate API because of this

Reply:
Agree. The interface is not written in stone and since its not widely used yet we can roll with this approach and update it with v2 when we'll get more feedback